tests/test_research_comprehensive.py

```
@@ -1,385 +1,259 @@
     print("PASS Data validation tests passed!")
-    return True
 
 
-def test_cli_integration() -> bool:
+def test_cli_integration() -> None:
     """Test CLI integration."""
     print("\n=== Testing CLI Integration ===")
 
-    try:
-        from cve_research_toolkit.cli import cli_main, main_research as main
+    from odin.cli import cli_main, main_research as main
```

Ruff (0.11.9)

`odin.cli.cli_main` imported but unused

Remove unused import: `odin.cli.cli_main`

(F401)